internal/providers/terraform/hcl_provider.go

```diff
@@ -573,12 +573,17 @@ func (p *HCLProvider) getResourceOutput(block *hcl.Block) ResourceOutput {
 
 	var configuration ResourceData
 	if block.HasModuleBlock() {
+		providerConfigKey = block.Provider()
+		alias := hcl.GetFromProvider(block, block.Provider(), "alias")
+		if alias.Type().Equals(cty.String) && alias.AsString() != "" {
+			providerConfigKey += "." + alias.AsString()
+		}
 		configuration = ResourceData{
 			Address:           stripCountOrForEach(block.LocalName()),
 			Mode:              "managed",
 			Type:              block.TypeLabel(),
 			Name:              stripCountOrForEach(block.NameLabel()),
-			ProviderConfigKey: block.ModuleName() + ":" + block.Provider(),
```

Comment:
I'm trying to find in Terraform docs what's the significance of the `:` separator for these modules. It seems that when there's a required terraform block in a sub module it prepends the module name, e.g. `module_1:aws` or `module_1:auth0`. I don't think it's a major issue, but can you put a comment above this line just to mark that we looked at this (in case it causes a bug in the future)

Reply:
i'll add the prefix back in, i removed it by mistake.